chapters/BP_015_en.md

```diff
@@ -39,7 +39,7 @@ the user experience more fluid.
 
 ### Example
 
-A seldomly updated display site about an enterprise with a contact form should rely on static web pages.
+An institutional seldomly updated display site about an enterprise with a contact form should rely on static web pages.
```

Comment:
I known the word is in the french version but the practice is interesting for any display site.